posthog/settings/dynamic_settings.py

```diff
@@ -109,6 +109,11 @@
         bool,
     ),
     "INGESTION_SITE_URL": (None, "Used in ingestion pipeline to determine sites url", str),
+    "CAPTURE_BACKEND_EXCEPTIONS": (
```

Comment:
Why setting vs defaulting to logging this?

Reply:
Good question. I assumed we weren't already logging everything because of concerns around data usage & log spew. But if that isn't the case, would be nice to just log all exceptions anyway.

Reply:
I would say unless there's important counter-arguments let's log everything. cc @guidoiaquinti and @hazzadous for their thoughts.

Reply:
There shouldn't be concerns around data usage afaik. If there is an exception, we should log it. If those exception logs are too many for the logging infra, then we have bigger issues.

There are some very noisy logs, e.g. the deprecated field logs but that's a separate issue.

Reply:
Sounds good, will do it like this then!